Blizzard.IO.RabbitMQ.Tests/NetqRabbitPublisherTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Moq;
+using NUnit.Framework;
+
+namespace Blizzard.IO.RabbitMQ.Tests
+{
+    [TestFixture]
+    class NetqRabbitPublisherTests
+    {
+        private Mock<IBus> _busMock;
+        private Mock<Core.ISerializer<string>> _serializerMock;
+        private Mock<IAdvancedBus> _advancedBusMock;
+        private Mock<IExchange> _exchangeMock;
+
+        private RabbitExchange _defaultDestinationExchange;
+        private byte[] _defaultSerializedData;
+        private string _defaultRoutingKey;
+        private string _defaultData;
+        private bool _defaultIsAbstract;
+
+        private NetqRabbitPublisher<string> _netqRabbitPublisher;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _busMock = new Mock<IBus>();
+            _serializerMock = new Mock<Core.ISerializer<string>>();
+            _advancedBusMock = new Mock<IAdvancedBus>();
+            _exchangeMock = new Mock<IExchange>();
+
+            _defaultSerializedData = new byte[0];
+            _defaultDestinationExchange = new RabbitExchange()
+            {
+                Name = "test",
+                Type = RabbitExchangeType.Fanout,
+                AlternateExchange = "test",
```

Comment:
Why constructing this propery?

Reply:
If i dont do it the string will get the default value of null and then in the Ctor of RabbitPublisher it will crash

Reply:
I suggest you check why this happens, we supposed to be able to construct an exchange with just only exchange type and name

Reply:
f